pages/guides/how-to-self-host-hermes.mdx

```diff
@@ -0,0 +1,224 @@
+# How to self host Hermes
+
+This guide explains how to self-host [Hermes](../documentation/pythnet-price-feeds/hermes.mdx). Hermes is Pyth Network's permissionless API server for Pyth price updates.
+It connects to both Pythnet and Wormhole to provide verified price update data for cross-chain usage. Please refer to the
+[Pyth cross-chain documentation](https://pyth-network.gitbook.io/pyth-network/pyth-cross-chain) for more information on how Pyth works.
+
+To run hermes you need to have access to a Pythnet RPC or run your own. This guide will explain how to run your own Pythnet RPC but it is recommended to use a node provider for more reliability and availability.
+You can use the following node providers to get access to a Pythnet RPC:
+
+- [Triton](https://triton.one)
+- [P2P](https://p2p.org)
+- [Blockdaemon](https://blockdaemon.com/)
+
+## Self-hosting on Kubernetes
+
+This section explains how to self-host Hermes on Kubernetes. It assumes that you have a basic understanding of Kubernetes and Helm. If you are new to Kubernetes, you can use the
+following resources to get started:
+
+- [Kubernetes Basics](https://kubernetes.io/docs/tutorials/kubernetes-basics/)
+- [Helm Quickstart](https://helm.sh/docs/intro/quickstart/)
+
+### Requirements
+
+- A running Kubernetes cluster: You can use [microk8s](https://microk8s.io/) or [minikube](https://minikube.sigs.k8s.io/docs/) to run a single node cluster on production machine.
```

Comment:
I'm hesitant in recommending microk8s in a production environment (zero experience with it), but minikube is definitely not recommended. I think k3s (k3s.io) is a better fit.